src/protocol/mod.rs

```diff
@@ -131,55 +125,6 @@ pub(crate) async fn read_lp(
     Ok(Some(buffer.split_to(size).freeze()))
 }
 
-/// A token used to authenticate a handshake.
-///
-/// The token has a printable representation which can be serialised using [`Display`] and
-/// deserialised using [`FromStr`].
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Deserialize, Serialize, MaxSize)]
-pub struct AuthToken {
```

Comment:
I guess we can dig this out again when needing the auth token in the delta chat integration...